core/src/main/java/tech/ydb/core/RetryPolicy.java

```diff
@@ -0,0 +1,22 @@
+package tech.ydb.core;
+
+/**
+ * Abstracts the policy to use when retrying some actions
+ *
+ * @author Aleksandr Gorshenin
+ */
+public interface RetryPolicy {
+    /**
+     * Called when an operation has failed for some reason. This method may return next values
+     * <ul>
```

Comment:
This is better be moved to @return section IMHO
The title could be extended to describe the purpose of this method. Something like this:
"Called when an operation is failed for some reason to determine if it should be retried. And if so, returns the delay to make the next retry attempt after"